packages/theme-default/src/col.css

```diff
@@ -7,20 +7,33 @@
 }
 
 @for $i from 1 to 24 {
-  .el-col-$i,
+  .el-col-$i {
+    width: calc(1 / 24 * $(i) * 100)%;
+  }
+  .el-col-offset-$i {
+    margin-left: calc(1 / 24 * $(i) * 100)%;
+  }
+  .el-col-pull-$i {
+    position: relative;
+    right: calc(1 / 24 * $(i) * 100)%;
+  }
+  .el-col-push-$i {
+    position: relative;
+    left: calc(1 / 24 * $(i) * 100)%;
+  }
+}
+
+@for $i from 1 to 24 {
```

Comment:
need to be wrap in media query

@media (max-width: 768px) {
  @for $i from 1 to 24 {
  }
}